tests/rptest/scale_tests/many_partitions_test.py

```diff
@@ -307,6 +307,7 @@ def __init__(self, test_ctx, *args, **kwargs):
                 # to pad out tiered storage metadata, we don't want them to
                 # get merged together.
                 'cloud_storage_enable_segment_merging': False,
+                'fetch_reads_debounce_timeout': 10,
```

Comment:
My concern here would be that we get the test to pass with this non-default configuration, but in general this configuration will be at default, and we should be testing the defaults unless we have a very good reason not to.

The test failing is a pretty good reason not to, I guess - but do we think this is the long term solution?

Reply:
> The test failing is a pretty good reason not to, I guess - but do we think this is the long term solution?

I think the long term solution will be to make `fetch_reads_debounce_timeout` be adjusted dynamically as discussed. That should also make the question of the extra fetch scheduling group less important as lots of the fetch cpu util will just go away.

 > My concern here would be that we get the test to pass with this non-default configuration, but in general this configuration will be at default, and we should be testing the defaults unless we have a very good reason not to.

I do obviously fully agree with this but to make the test stable for now we do need to do one thing and I think increasing the debounce time is conceptually less of a difference to the defaults than just disabling the extra fetch group.